modules/caddyhttp/fileserver/staticfiles.go

```diff
@@ -153,6 +153,16 @@ type FileServer struct {
 	// a 404 error. By default, this is false (disabled).
 	PassThru bool `json:"pass_thru,omitempty"`
 
+	// Override the default sort. It includes the following options:
+	// sort_by: name(default), size, time
+	// order: asc(default), desc
+	// eg.:
+	//   - `sort time desc` will sort by time in descending order
+	//   - `sort size` will sort by size in ascending order
+	//   - `sort` will sort by name in ascending order, which is the default
+	// The order of the sort options is not important.
+	SortOptions []string `json:"sort,omitempty"`
```

Comment:
Since this is a freeform slice of strings, we should probably validate this input at `Provision()`-time. It should be documented (and enforced) that only 1 or 2 values are allowed: the first must be a field name to order by, and the second if present should be the direction.

Reply:
Thanks for the reminder, or I wouldn't have known about Provision().